airflow/www/auth.py

```diff
@@ -44,6 +46,27 @@ def get_access_denied_message():
     return conf.get("webserver", "access_denied_message")
 
 
+def has_access(permissions: Sequence[tuple[str, str]] | None = None) -> Callable[[T], T]:
+    """
+    Factory for decorator that checks current user's permissions against required permissions.
+
+    Deprecated. Do not use this decorator, use one of the decorator `has_access_cluster_*` defined in
+    airflow/www/auth.py instead.
```

Comment:
Maybe be a bit more explicit here?
```suggestion
    Deprecated. Do not use this decorator, use one of the decorator `has_access_cluster_*` defined in
    airflow/www/auth.py instead.
    This decorator will only work with FAB authentication and not with other auth providers.
```

Reply:
I like it